src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -722,6 +726,21 @@ private static bool IsTypeLessVisibleThan(TypeSymbol type, Symbol sym, ref Compo
                     }
                 }
 
+                if (visitCustomModifiers && typeWithAnnotationsOpt.HasType)
+                {
+                    foreach (var customModifier in typeWithAnnotationsOpt.CustomModifiers)
+                    {
+                        var result = VisitType(
+                            typeWithAnnotationsOpt: default, type: (NamedTypeSymbol)customModifier.Modifier,
+                            typeWithAnnotationsPredicate, typePredicate, arg,
```

Comment:
>typePredicate [](http://example.com/codeflow?start=58&length=13)

I guess at the top we also want to assert that ```typePredicate``` is not null when ```visitCustomModifiers``` is true.

Reply:
Sure.